priv/site/_includes/mobile_nav.slime

```diff
@@ -1,4 +1,4 @@
-.MobileNav style="--text-color: var(--c-<%= @env[:color] || "white" %>)"
+.MobileNav style="--text-color: var(--c-<%= @env[:color] || "white" %>); border-color: var(--c-<%= @env[:color] || "white" %>)"
```

Comment:
This feels odd to me, is there any specific reason to do this inline vs having a css rule using the variable with a fallback to `white`?